pkg/namespace/cmd.go

```diff
@@ -77,14 +77,14 @@ func (c cmd) Modify(ns Namespace, b *Builder) error {
 		}
 		return ns.Mount(servername, old, spec, c.flag)
 	case UNMOUNT:
-		new, old := "", ""
+		newNS, oldNS := "", ""
 		if len(args) == 2 {
-			new = args[0]
-			old = args[1]
+			newNS = args[0]
```

Comment:
here you could use the form from above, makes searches easier, 
newNS, oldNS = args[0], args[1]